charts/artifact-hub/values.schema.json

```diff
@@ -216,6 +216,11 @@
                             "type": "boolean",
                             "default": false
                         },
+                        "cacheDir": {
+                            "title": "If set, the cache directory for the Helm client will be explicitly set (otherwise defaults to $HOME/.cache), and the directory will be mounted as ephemeral volume (emptyDir).",
```

Comment:
Could you use the current `title` value as the `description` field and pick a shorter title? It will look much nicer when rendered on the values schema modal :)

Reply:
Done.